src/lsp/cobol_parser/cobol_parser.ml

```diff
@@ -21,6 +21,9 @@ module Outputs = Parser_outputs
 module Diagnostics = Parser_diagnostics
 
 module Tokens = Grammar_tokens
+module Recover = Grammar_recover
```

Comment:
```ocaml
      let default_value =
        try Some (Cobol_parser.Recover.default_value sym)
        with Not_found -> try Some (Expect.guessed_default_value_of_nullables sym)
        with Not_found -> if debug then  Lsp_io.log_debug "Not found"; None in
```
The default_value for `Menhir.feed`
I guess I can put everything in `guessed_default_value` but that would cause duplicate code

Reply:
If `sym` is always a nonterminal, I think it's also better to avoid generalizing to a symbol.